spec/rails_helper.rb

```diff
@@ -43,10 +43,3 @@ def login(user)
 
   click_on "Login"
 end
-
-Shoulda::Matchers.configure do |config|
-  config.integrate do |with|
-    with.test_framework :rspec
-    with.library :rails
-  end
-end
```

Comment:
Yeah.  For some reason we had two copies of the Shoulda:Matcher set up in there.  I thing it happened following a merge conflict fix.